Slider/themes/holodark/Slider.css

```diff
@@ -16,13 +15,104 @@
 .d-slider input {
   display: none;
 }
+.d-slider .d-slider-bar {
+  border: 1px inset #cccccc;
+  border-radius: 4px;
+  padding: 0;
+  margin: 0;
+  background-clip: content-box;
+  background-color: #f5f5f5;
+}
+.d-slider .d-slider-container {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  position: relative;
+  height: 100%;
+  width: 100%;
+  box-shadow: 0 0.1em 0.2em 0.1em rgba(0, 0, 0, 0.1) inset;
+}
+.d-slider .d-slider-progress-bar {
+  -webkit-box-sizing: content-box;
+  -moz-box-sizing: content-box;
+  box-sizing: content-box;
+  border-width: 0;
+  background-color: #999999;
+}
+.d-slider.d-disabled {
+  cursor: default;
+  -webkit-box-shadow: none;
+  -moz-box-shadow: none;
+  box-shadow: none;
+}
+.d-slider.d-disabled .d-slider-container {
+  -webkit-box-shadow: none;
+  -moz-box-shadow: none;
+  box-shadow: none;
+  border-color: #ffffff;
+}
+.d-slider.d-disabled .d-slider-progress-bar {
+  background: none;
+}
+.d-slider.d-disabled .d-slider-handle,
+.d-slider.d-disabled .d-slider-handle:hover,
+.d-slider.d-disabled .d-slider-handle:focus,
+.d-slider.d-disabled .d-slider-handle:active,
+.d-slider.d-disabled .d-slider-handle.active {
+  border-color: #cccccc;
+  -webkit-box-shadow: none;
+  -moz-box-shadow: none;
+  box-shadow: none;
+  background-color: #f5f5f5;
+}
+.d-slider-container > * {
+  position: absolute;
+}
+.d-slider-handle {
+  user-select: none;
+  -webkit-user-select: none;
+  -ms-user-select: none;
+  -moz-user-select: none;
+  position: absolute;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  border: 1px outset #999999;
+  border-radius: 6px;
+  padding: 0;
+  margin: 0;
+  background-clip: content-box;
+  background-color: #f5f5f5;
+  z-index: 1;
+}
+.d-slider-handle:focus,
+.d-slider-handle:active:focus,
+.d-slider-handle.active:focus {
+  outline: thin dotted;
+  outline-offset: -3px;
+}
+.d-slider-handle:hover,
+.d-slider-handle:focus {
+  color: #333333;
+  text-decoration: none;
+}
+.d-slider-handle:focus + .d-slider-handle {
+  z-index: 0;
+}
+.d-slider-handle:active,
+.d-slider-handle.active {
+  outline: 0;
+  background-image: none;
+  -webkit-box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+  -moz-box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+  box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+}
 .d-slider-h {
   width: 200px;
 }
 .d-slider-h .d-slider-container {
-  width: 100%;
   min-width: 80px;
-  height: 5px;
+  height: 8px;
```

Comment:
I replaced holodark and ios themes by bootstrap to save time. more infos in the commit:https://github.com/dmandrioli/deliteful/commit/f6589013f2389eb98e546ecfc68961fe54fcda05

This is a voluntary regression, fixing it would require some time since the less/css refactoring is quite important in this commit. The dedicated PR is https://github.com/ibm-js/deliteful/pull/480/files